lib/prometheus-metrics.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
```

Comment:
I think you should put the `require()` statement here:

```suggestion
const client = require('prom-client');
```

Reply:
Probably yes, but the fact is the when we `require('prom-client')`  it automatically starts to collect the metrics, and I thought that was the reason for the errors. But seems the errors are related to webpack config which still under investigation.

Reply:
>  when we require('prom-client') it automatically starts to collect the metrics

I don't think this is actually the case. The metrics don't start being collected until `collectDefaultMetrics()` is called, or you provide a custom metric to the module.